news/4576.hooks.rst

```diff
@@ -0,0 +1 @@
+Add hook for jaraco.text
```

Comment:
Please also *copy* this file to 4632.… to make the release process create the appropriate reference.